src/js/models/layer.js

```diff
@@ -134,12 +134,12 @@ define(function (require, exports, module) {
         /**
          * @type {Immutable.List.<Shadow>}
          */
```

Comment:
I'm fine with initializing immutable record members like this to empty lists. Immutable is smart enough to only ever create a single empty list object, so there should be no performance impact. I would be _more_ concerned with making such a change consistently throughout our models, and also ensuring that client code no longer performs unnecessary falsey checks.